data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,143 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Part 1",
+  "extent": ["f. 1-152"],
+  "dimensions": [{
+    "type": "average-folio",
+    "value": "320 x 250 x 92 mm"
+  }],
+  "support": {
+    "type": "Parchment",
+    "features": [],
+    "note": ["Quire 20 is paper."]
+  },
+  "condition": {
+    "note": [],
+    "features": []
+  },
+  "writing_and_hands": [{
+    "language": "Syriac",
+    "writing_system": "Syriac",
+    "script": "Estrangela",
+    "ink_color": "black",
+    "hand_note": ["Taw, Alaph, and Dalath/Resh all exhibit features of Serto."],
+    "textual_artifacts": ["ark:/21198/z1x07bdf"],
+    "extent": ["f. 1-152"]
+  }],
+  "page_layout": [{
+    "columns": 1,
+    "number_of_lines": 26,
+    "column_width": "",
+    "margin_width": "",
+    "features": [],
```

Comment:
e.g., ‘Ruling’. https://jira.library.ucla.edu/browse/SLDP-632 is for sorting out our full list of available features, where we allow feature fields, and what the various subsets for each of those locations should be.